doc/example/example_Fujita/Fujita_experimentalCondition.tsv

```diff
@@ -0,0 +1,7 @@
+conditionId	conditionName	EGF
```

Comment:
When including test models in petab itself, we have to be careful when the respective model or format is updated in the benchmark collection. But I hink that it makes sense to have some test models in petab indeed.

Reply:
The point is:
If I want to do write some code, I want to be able to run the most important tests also if I am offline (maybe in a train, maybe at home and have an internet failure, maybe Wi-Fi is down, ...).
This is why I just added the files of those two examples (Isensee and Fujita) locally to PEtab. Obviously, one could think of making up simpler test cases, which needs less time to run.
However, right now a visualization tests takes ~20 seconds, which I still consider fine...

Reply:
Yes, should be fine :+1: